resources/assets/js/src/components/alert/Alert.tsx

```diff
@@ -11,10 +12,10 @@ const Alert: any = (props) => {
             text = "Neznámá chyba";
```

Comment:
Typo in line 8 přiLAŠovací, fix to přihlašovací. You may use english for a now in whole application. We will deal with translations right after.